### PR TITLE
Decode proposal data

### LINKED ISSUE #650: Decode proposal data to display raw text

### PR DESCRIPTION
**Pull request by AhmedKorim**
## Summary of changes
- Decode proposal data in the details page for the proposal data

### Reference issue to close (if applicable)
- Closes #650 

-----
### Code Checklist 

- [x] Tested
- [x] Documented

### CODE REVIEW

**Comment by AhmedKorim** [comment]
This still missing
1. `AnchorCreateProposal`
2. `EvmProposal`
3. `ProposerSetUpdateProposal`
4. `RefreshVote`

**Review by AtelyPham — COMMENTED** [review]
Look good! But I think it would be better if we move the file `getProposalText.ts` into the `packages/page-statistics/src/utils` instead of `packages/page-statistics/src/containers/ProposalDetail/getProposalText.ts`. And do you think `getProposalDataByProposalType` is a better name for this function 🤔  @AhmedKorim

**Comment by dutterbutter** [comment]
@AhmedKorim please resolve conflicts

**Comment by AhmedKorim** [comment]
This pr waits for https://github.com/webb-tools/webb.js/pull/297

**Comment by AhmedKorim** [comment]
This Pr was updated to the latest webb.js with the messing proposals and it errors now
image

**Comment by dutterbutter** [comment]
@AhmedKorim please resolve conflicts here

File: libs/polkadot-api-provider/src/webb-provider/vanchor-withdraw.ts [hunk 1]
```diff
@@ -362,9 +367,9 @@ export class PolkadotVAnchorWithdraw extends VAnchorWithdraw<WebbPolkadot> {
       this.cancelToken.throwIfCancel();
       txHash = await tx.call(account.address);
     }
-
+    // TODO construct the right note
```
Comment (nepoche) [inline libs/polkadot-api-provider/src/webb-provider/vanchor-withdraw.ts:370]: What is wrong with the note?

**Review by nepoche — CHANGES_REQUESTED** [review]

File: apps/stats-dapp/src/utils/getProposalsData.ts [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+  TokenAddProposal,
+  TokenRemoveProposal,
+  WrappingFeeUpdateProposal,
+} from '@webb-tools/sdk-core';
+
+import { hexToU8a, u8aToHex } from '@polkadot/util';
+export function getProposalsData(
+  propType: ProposalType,
+  data: string
+): Record<string, string | Record<string, any>> {
```
[the remark this answers is not in the document]
Comment (AhmedKorim) [reply apps/stats-dapp/src/utils/getProposalsData.ts:24]: it's just raw json

File: libs/polkadot-api-provider/src/webb-provider/vanchor-withdraw.ts (hunk 1 above)
Comment (AhmedKorim) [reply libs/polkadot-api-provider/src/webb-provider/vanchor-withdraw.ts:370]: This was from a previous change will remove the comment

**Review by nepoche — APPROVED** [review]

**Pull request merged**